setup.py

```diff
@@ -1,11 +1,41 @@
-#!/usr/bin/env python
+from setuptools import setup
 
-from distutils.core import setup
-from catkin_pkg.python_setup import generate_distutils_setup
+package_name = 'urdfdom_py'
 
-d = generate_distutils_setup(
+setup(
+    name=package_name,
+    version='0.3.3',
```

Comment:
melodic-devel is currently on 0.4.0, but since this is going to be a new branch, we probably want to bump up to something like 1.0.0 here (and in the package.xml).